addons/aircraft/functions/fnc_initEjectAction.sqf

```diff
@@ -0,0 +1,44 @@
+/*
+ * Author: Dystopian
+ * Add Eject action to vehicle if needed.
+ *
+ * Arguments:
+ * 0: Vehicle <OBJECT>
+ *
+ * Return Value:
+ * None
+ *
+ * Example:
+ * _vehicle call ace_aircraft_fnc_initEjectAction
+ *
+ * Public: No
+ */
+#include "script_component.hpp"
+
+params ["_vehicle"];
+
+if (unitIsUAV _vehicle) exitWith {};
+
+private _config = configFile >> "CfgVehicles" >> (typeOf _vehicle);
+
+private _addAction = false;
+
+if (0 == getNumber (_config >> "driverCanEject")) then {
+    _vehicle setVariable [QUOTE(DOUBLES(GVAR(ejectAction),driver_[])), true];
+    _addAction = true;
+};
+
+{
+    _x params ["", "_role", "", "_turretPath"];
+    if (0 == getNumber (([_config, _turretPath] call CBA_fnc_getTurret) >> "canEject")) then {
+        _vehicle setVariable [format ["%1_%2_%3", QGVAR(ejectAction), _role, _turretPath], true];
+        _addAction = true;
+    };
+} forEach (fullCrew [_vehicle, "gunner", true] + fullCrew [_vehicle, "turret", true]);
```

Comment:
commander?

Reply:
There are no `Air` vehicles ATM in vanilla + RHS + CUP with `commander` in `fullCrew` (just checked). Copilot is gunner. And I think there will not.

Reply:
I think we should still add it, it can't hurt, right?

Reply:
sure, why not.